examples/data-objects/key-value-cache/src/index.ts

```diff
@@ -118,16 +118,14 @@ export class KeyValueFactoryComponent implements IRuntimeFactory, IFluidDataStor
         const mapFactory = SharedMap.getFactory();
         dataTypes.set(mapFactory.type, mapFactory);
 
-        const runtime = FluidDataStoreRuntime.load(
-            context,
-            dataTypes,
-        );
+        const runtimeFactory = mixinRequestHandler(
```

Comment:
(Asking mostly because I'm not super familiar with conventions around dynamically creating classes)
Is there a different naming convention for these created classes (e.g. named like actual classes starting with a capital letter), or are they named like regular variables?  `runtimeFactory` in particular sounds a bit unusual to me because of how it's used; `new runtimeFactory` sounds like it would create a new runtime factory rather than a new runtime.  Something like `runtimeClass` would make more sense imo